### PR TITLE
rewrites test app to ES6

### PR DESCRIPTION
**Pull request by DominikGuzei**
Rewrites the test app to ES6 + introduces a simple namespace for it.

### CODE REVIEW

File: source/server/module-automation.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ Space.Module.mixin({
   },
 
   _onInitializeEventSourcing: function(onInitialize) {
-    var module = this;
+    let module = this;
```
Comment (rhyslbw) [inline source/server/module-automation.js]: we should be using `const` as the default, unless overriding is intended right?

Comment (DominikGuzei) [inline source/server/module-automation.js]: Yeah we can do that :wink:

**Comment by rhyslbw** [comment]
Other than the inline comment, this looks great, well done for smashing this not so fun task!

**Comment by rhyslbw** [comment]
Closing to re-open against _develop_ (no diff, so no merging required)

**Pull request closed without merging**